js/build.js

```diff
@@ -10,9 +10,16 @@ Fliplet.Widget.instance('chart-pie-1-1-0', function(data) {
   if (themeInstance) {
     Object.assign({}, themeInstance.data.values);
 
+    if(!themeValues) {
+      var themeValue = themeInstance.data.values;
```

Comment:
Declaration of the variable should occur before the `if`. Otherwise from the way the code reads, there's no way `themeValues` would have a value.

Reply:
@tonytlwu, declaration of `themeValues` is already at [this](https://github.com/Fliplet/fliplet-widget-chart-pie/blob/master/js/build.js#L8) line before `if` condition.

Please provide your inputs on this.

Reply:
@tonytlwu, both are different variables, `themeValues` and `themeValue`. And in `if `condition we are checking `themeValues` which we declared previously.

Reply:
@NikitaJadhavXoriant I see. I didn't spot the difference with the `s`. In that case this is fine.